src/read.c

```diff
@@ -4544,7 +4541,11 @@ static avifResult avifDecoderFindGainMapItem(const avifDecoder * decoder,
         }
 
         const avifProperty * pixiProp = avifPropertyArrayFind(&toneMappedImageItemTmp->properties, "pixi");
-        if (pixiProp && pixiProp->u.pixi.planeCount > 0) {
+        if (pixiProp) {
+            if (pixiProp->u.pixi.planeCount == 0) {
+                avifDiagnosticsPrintf(data->diag, "Box[pixi] of tmap item contains invalid plane count [%u]", pixiProp->u.pixi.planeCount);
```

Comment:
Note that the specification does not forbid `num_channels` to be 0. I meant it as libavif should ignore or reject an item tagged with such as property.  
The current error message is fine though.